Shipr-Containerization/shipr-payment/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.9-slim
+
+WORKDIR /app
+
+COPY requirements.txt /app/requirements.txt
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /app
+
+CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8000"]

```

Comment:
Please avoid using root user in all your Dockerfiles.

Reply:
Done